server/chat-commands/moderation.ts

```diff
@@ -1629,14 +1629,15 @@ export const commands: ChatCommands = {
 	hidelines: 'hidetext',
 	hlines: 'hidetext',
 	cleartext: 'hidetext',
+	ctext: 'hidetext',
 	clearaltstext: 'hidetext',
 	clearlines: 'hidetext',
 	forcecleartext: 'hidetext',
 	hidetext(target, room, user, connection, cmd) {
 		if (!target) return this.parse(`/help hidetext`);
 		room = this.requireRoom();
 		const hasLineCount = cmd.includes('lines');
-		const hideRevealButton = cmd.includes('clear');
+		const hideRevealButton = cmd.includes('clear') || cmd.includes('c');
```

Comment:
```suggestion
		const hideRevealButton = cmd.includes('clear') || cmd === 'ctext';
```
Otherwise, any `force` command (e.g. `/forcehidetext`) would hide the button.